Doc/library/symtable.rst

```diff
@@ -182,10 +184,42 @@ Examining Symbol Tables
       Return ``True`` if the symbol is referenced in its block, but not assigned
       to.
 
+   .. method:: is_free_class()
+
+      Return *True* if a class-scoped symbol is free from
```

Comment:
Though strictly speaking this can also get used for symbols in annotation scopes:

```
def f():
    x = 1  # function-scoped
    class C:
        x = 2
        type Alias = (x, lambda: x)
```


```
            symbol table for nested type alias 'Alias':
                free symbol '__classdict__': use
                global_implicit symbol 'x': use, def_free_class

                symbol table for nested function 'lambda':
                    free symbol 'x': use
```

But I think it's still fair to say "class-scoped symbol" here, as the symbol is inherited from the class scope.

Reply:
If we go into that direction, I don't know how to name that 'x' inside... "class-scoped" seemed clear enough IMO and if we add something like "inherited class-scoped symbols in annotation scopes", I feel it could be more confusing (unless we add an example).